dev/PushNotifications/PushNotificationManager.cpp

```diff
@@ -11,15 +11,19 @@
 
 #include <winerror.h>
 #include <algorithm>
-#include <vector>
 #include "PushNotificationChannel.h"
 #include "externs.h"
 
 constexpr PCWSTR backgroundTaskName = L"PushBackgroundTaskName";
 
-using namespace winrt::Windows::ApplicationModel::Background;
-using namespace winrt::Windows::Networking::PushNotifications;
-using namespace winrt;
+namespace ReunionPushNotifications = winrt::Microsoft::Windows::PushNotifications;
```

Comment:
We shouldn't be using Reunion in our code. Can't you move this to namespace winrt below?

Reply:
The issue is that including namespace Windows::Networking::PushNotifications and winrt::Microsoft::Windows::PushNotifications creates an ambiguous type when using winrt::* (anything PushNotifications).

Reply:
For now, lets just use the full form if there are namespace collisions.